CHANGELOG.md

```diff
@@ -28,15 +40,15 @@
 **Note: this release has a critical issue and was deprecated. Please update to 15.5.4 or higher.**
 
 ### React
-* Fix erroneous PropTypes access warning. ([@acdlite](https://github.com/acdlite) in ([ec97ebb](https://github.com/facebook/react/commit/ec97ebbe7f15b58ae2f1323df39d06f119873344))
+* Fix erroneous PropTypes access warning. ([@acdlite](https://github.com/acdlite) in ([#ec97ebb](https://github.com/facebook/react/commit/ec97ebbe7f15b58ae2f1323df39d06f119873344))
```

Comment:
Sorry I wasn't being clear. We put `#` in for PR numbers, but don't include it for commits.